source/install/desktop-managed-resources.rst

```diff
@@ -0,0 +1,38 @@
+
+Desktop - Managed Resources
+===========================
+
+This guide provides steps to use managed resources with the Mattermost Desktop App. A managed resource can be any service available on the same hostname using the same protocol as the Mattermost server.
```

Comment:
Will steps be added to the guide in the next iteration?

In the interim, one option could be: 

```suggestion
The Mattermost Desktop App supports managed resources. A managed resource can be any service available on the same hostname using the same protocol as the Mattermost server.
```